src/mysql-metrics/Gopkg.toml

```diff
@@ -19,36 +19,3 @@
 # [[override]]
 #  name = "github.com/x/y"
 #  version = "2.4.0"
-
-
-[[constraint]]
-  branch = "master"
-  name = "code.cloudfoundry.org/lager"
-
-[[constraint]]
-  name = "github.com/DATA-DOG/go-sqlmock"
-  version = "1.3.0"
-
-[[constraint]]
-  name = "github.com/cloudfoundry/dropsonde"
-  version = "1.0.0"
-
-[[constraint]]
-  name = "github.com/go-sql-driver/mysql"
-  version = "1.3.0"
-
-[[constraint]]
-  branch = "master"
-  name = "github.com/hashicorp/go-multierror"
-
-[[constraint]]
```

Comment:
Ginkgo is still a direct dependency of this project right? We don't understand why we've removed it as a constraint.

Reply:
Yes, it's still in the Gopkg.lock, it looks like the constraint on version 1.4 just got removed, so that the project can use the latest Ginkgo.